src/firewall-applet.in

```diff
@@ -920,12 +922,17 @@ class TrayApplet(QtWidgets.QSystemTrayIcon):
             return
 
         if uuid:
-            if "kde-" in NM_CONNECTION_EDITOR:
+            if "systemsettings" in NM_CONNECTION_EDITOR:
+                os.system("%s kcm_networkmanagement --args Uuid=%s" % (NM_CONNECTION_EDITOR, uuid))
```

Comment:
Can you confirm this does NOT need the ampersand `&` ?

Reply:
Oh, It needs the ampersand `&` to create a new process. The main process doesn't have to wait for this process.

Reply:
Fixed

Reply:
I can't do git push, you can fix it as you know.

Reply:
I'll add it and push. I'm also squashing you commits and fixing the commit message.